frontend/src/main-components/StudyModeGameHandler.js

```diff
@@ -24,19 +30,24 @@ function nextQuizWord() {
             nextWord = quiz[currentArray].shift();
         }
     }
-    currentQuizWord = nextWord;
-    return nextWord;
+  currentQuizWord = nextWord;
+  return nextWord;
 }
 
-function updateWordQueues(correct) {
-    if (correct === "false" && currentArray < (quiz.length - 1)) {
-        quiz[currentArray + 1].push(currentQuizWord);
-    }
-    //TODO(esaaracay): Fetch /study to update familarity score
+function updateWordQueues(correct, cardKey) {
+  if (correct == "false" && currentArray < Quiz.length - 1) {
+    quiz[currentArray + 1].push(currentQuizWord);
+  }
+  // Updates the card's familarity score and stores it
+  fetch('/study', {
```

Comment:
Return this promise so the caller can await on it.

Reply:
done.